### ISSUE TITLE
DS - SnackBar Accessibility Review

### ISSUE THREAD

**Issue by kellylein** [department-of-veterans-affairs/va-mobile-library#411]
## Description 
As part of this story we want to:   

- Once the Snackbar component is available in Storybook, Brea will complete an accessibility review and create tickets for any additional work.

## Acceptance Criteria
  

-  [x] Setup and familiarize with the Storybook tool
-  [x] Review SnackBar component for and accessibility issues
-  [x] Create Bug tickets for any changes needed

**Comment by brea11y** [comment]
Hey @kellylein – I don't appear to have access / permissions within the VA Mobile Library board to check off the AC within the ticket or create new tickets.

There was only one thing that I noticed during testing, but this may not be an issue? When testing with Talkback (Android) and VoiceOver (iOS), when the snackbar appears, it has "with accessibility override" announced at the end. This appears to be attached to the copy, since it is announced when I navigate to the copy with the screenreaders, as well.

Everything else looks and sounds great!

**Comment by brea11y** [comment]
@kellylein – Could we check off the AC's here and mark this as complete? I don't seem to have permissions to make any changes to it.

**Comment by drjecker** [comment]
Talked to Brea and closing as she does not have permission.
